HACKING.rst

```diff
@@ -1,4 +1,35 @@
-The first time you start working in the repository, set up a clean test
+A) MOC supports both sqlite and postgresql. If you are using sqlite, you 
```

Comment:
HaaS. The docs shouldn't be talking about the MOC here; it isn't relevant to what the software is capable of.